### PR TITLE
Bugfix/some corrections to scheme adapter docs

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ A detailed documentation for dfsps who want to test the mojaloop cluster deploym
 ### Mojaloop Local K8S cluster deployment
 Please follow the below link to deploy your own cluster on local system.
 https://mojaloop.io/documentation/deployment-guide/
-MacOS is recommended and atleast 16GB Ram and 4 core processor is required.
+
+A Linux based operating system is recommended and atleast 16GB Ram and 4 core processor is required.
```
Comment (yosheeck) [inline mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md]: typo "atleas"

File: mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md [hunk 2]
```diff
@@ -119,3 +178,4 @@ curl -X POST \
 ```
 
 You should get a response with COMPLETED currentState.
+If you get
```
Comment (yosheeck) [inline mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md]: never finished sentence ?

File: mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md (hunk 1 above)
Comment (vijayg10) [reply mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md]: Changed

File: mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md (hunk 2 above)
Comment (vijayg10) [reply mojaloop-technical-overview/sdk-scheme-adapter/usage/scheme-adapter-and-local-k8s/README.md]: removed it, thanks

**Pull request merged**